Tests/KSCrashRecordingCoreTests/KSThread_Tests.m

```diff
@@ -35,36 +35,35 @@ @interface KSThread_Tests : XCTestCase @end
 
 @implementation KSThread_Tests
 
-// TODO: Disabling this until I figure out what's wrong with queue names.
```

Comment:
Not quite.
I was just looking through other pull requests and found this #221. Don't think that it would be compatible with my changes. By the way tests passes fine.